R/gen-namespace-docs.R

```diff
@@ -1967,6 +1967,40 @@ NULL
 #' @export
 NULL
 
+#' fftfreq
+#'
+#' Computes the discrete Fourier Transform sample frequencies for a signal of size `n`.
+#'
+#' @note 
+#' By convention, `fft()` returns positive frequency terms first, followed by the negative
+#' frequencies in reverse order, so that `f[-i]` for all `0 < i ≤n/2`
```

Comment:
The character `≤` here is probably causing an error in the CI:

```
❯ checking PDF version of manual ... [21s/21s] WARNING
  LaTeX errors when creating PDF version.
  This typically indicates Rd problems.
  LaTeX errors found:
  ! LaTeX Error: Unicode character ≤ (U+2264)
                 not set up for use with LaTeX.
```

Reply:
Thanks, so great you're already pointing this out! (I hadn't checked out the exact reason yet, because I wanted to fix other things first)